code/modules/projectiles/projectile/bullets.dm

```diff
@@ -34,16 +34,37 @@
 	damage = 15
 
 /obj/item/projectile/bullet/pellet/weak
+	damage = 6
+	New()
+		range = rand(8)
+
+/obj/item/projectile/bullet/pellet/weak/on_range()
+	var/datum/effect_system/spark_spread/sparks = new /datum/effect_system/spark_spread
+	sparks.set_up(1, 1, src)
+	sparks.start()
+	..()
+
+/obj/item/projectile/bullet/pellet/overload
 	damage = 3
+	New()
+		range = rand(10)
 
-/obj/item/projectile/bullet/pellet/random/New()
-	damage = rand(10)
+/obj/item/projectile/bullet/pellet/overload/on_hit(atom/target, blocked = 0)
+	..()
+	explosion(target, 0, 0, 2)

```

Comment:
Level as in punch holes through the tiles and damage wiring? Your issue might be the two negative values. I found that my problem with hullbreaches went away when I set my devastation and high damage values to 0.